samples/communication-between-nodes/src/main/java/com/badoo/ribs/samples/comms_nodes/rib/language_selector/Language.kt

```diff
@@ -0,0 +1,22 @@
+package com.badoo.ribs.samples.comms_nodes.rib.language_selector
+
+import android.os.Parcelable
+import com.badoo.ribs.android.text.Text
+import kotlinx.android.parcel.Parcelize
+
+sealed class Language : Parcelable {
```

Comment:
This class feels weird. You could do it simpler by just making it an enum, using the enum's toString where needed and wrapping it on to Text.Plain right there, instead of conflating them.